src/components/calcite-radio-button-group/calcite-radio-button-group.tsx

```diff
@@ -55,28 +46,10 @@ export class CalciteRadioButtonGroup {
   @Prop({ reflect: true }) required = false;
 
   /** The scale (size) of the radio button group. */
-  @Prop({ mutable: true, reflect: true }) scale: "s" | "m" | "l" = "m";
-
-  @Watch("scale")
-  validateScale(newScale: string) {
-    const scales = ["s", "m", "l"];
-    if (!scales.includes(newScale)) {
-      this.scale = "m";
-      this.passPropsToRadioButtons();
-    }
-  }
+  @Prop({ reflect: true }) scale: "s" | "m" | "l" = "m";
 
   /** The color theme of the radio button group. */
-  @Prop({ mutable: true, reflect: true }) theme: "light" | "dark" = "light";
-
-  @Watch("theme")
-  validateTheme(newTheme: string) {
-    const themes = ["light", "dark"];
-    if (!themes.includes(newTheme)) {
-      this.theme = "light";
-      this.passPropsToRadioButtons();
-    }
-  }
```

Comment:
Although I think we should keep both of these watchers, just remove the validation and call them something like `scaleWatcher` and `themeWatcher` because these will need to update children radio-buttons when those props change.

Reply:
🔟 – 4️⃣

Reply:
💥